aries-site/src/data/structures/Structure.js

```diff
@@ -0,0 +1,27 @@
+/* Utility functions to support page manipulations */
+
+export class Structure extends Array {
+  sortByAvailability() {
+    const availability = { true: 0, false: 1 };
+
+    return this.sort(
+      (a, b) => availability[a.available] - availability[b.available],
+    );
+  }
+
+  sortByCardOrder() {
+    return this.sort((a, b) => a.cardOrder - b.cardOrder);
+  }
+
+  sortByCategory(weights) {
+    return this.sort((a, b) => weights[a.category] - weights[b.category]);
+  }
+
+  sortByName() {
```

Comment:
Is this alphabetizing? If so, can we just do `this.sort()` since sort by default will alphabetize?

Reply:
For an array of strings we could. Since we are sorting objects by a property, I think we have to go long form, right?

Reply:
Ahh, yes I think you're right there